crypto/data_formats.go

```diff
@@ -15,41 +15,68 @@
 package crypto
 
 import (
-	"encoding/base64"
 	"fmt"
-	"strconv"
 
-	"github.com/benlaurie/objecthash/go/objecthash"
+	"github.com/google/certificate-transparency-go/tls"
```

Comment:
ooo - this is a bit of an unfortunate dependency :/

Reply:
Why is it unfortunate?

Reply:
Because c-t-go takes a dependency on Trillian, which will have a dependency on c-t-go.
Though I guess this is just implying that the `tls` package should probably live elsewhere.